### ISSUE TITLE
Accessibility: MAS05A: Keyboard: Keyboard focus is not properly visible for the next and previous buttons.

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#36]
#A11yMAS
#MAS40B
#Wipro
#vscode-website
#FireFox

Testing Environment:
OS-RS1
OS Name Microsoft -Windows 10 Pro
Version-10.0.14393 Build 14393.607
Browser-Edge
Edge Version-Microsoft Edge 38.14393.607.0
Firefox Version:50.1.0

**Repro Steps:**
1. Open [https://code.visualstudio.com/](https://code.visualstudio.com/) in edge.
2. Navigate to the  "Next"  button above "Meet IntelliSense." in middle of the page.[See screenshot]
3. Observe keyboard focus [Bounded rectangular box]

**Actual Result:**
Keyboard focus [Bounded rectangular box] is not visible  for the "next" and "previous" buttons above the "Meet IntelliSense." in middle of the page.

**Expected Result:**
Keyboard focus [Bounded rectangular box] is should be visible  for the "next" and "previous" buttons above the "Meet IntelliSense." in middle of the page.

**User Impact:**
Users with visual and/or motor impairments who rely on keyboard may not be able to identify that the element is focused or not.

**Screenshots:**
![inpect view_next button](https://cloud.githubusercontent.com/assets/24825441/21710186/13f373c0-d40d-11e6-8caf-57a1f6a508f0.png)
![no visible rectangular box for buttons](https://cloud.githubusercontent.com/assets/24825441/21710187/13f7ab52-d40d-11e6-822b-7a6101bdc6cd.png)

**Comment by chrisdias** [comment]
Similar to https://github.com/wiproaccess/vscode-website/issues/42 and I can access the Next/Prev buttons.
